tests/particle/test_particle.py

```diff
@@ -23,15 +23,16 @@
 
 def test_find():
     # 1 match found
-    prepr = repr(Particle.find(name="gamma"))
+    with pytest.deprecated_call():
+        prepr = repr(Particle.find(name="gamma"))
```

Comment:
You should always check for warnings (with `pytest.warns`) and deprecation calls. PyTest logs should be clean, and you do want to ensure these are producing the expected warnings. Technically, this is probably producing the wrong type of deprecation call - a user facing deprecation call should use `category=FutureWarning`, but that can be fixed in a followup PR.

Reply:
Thanks. Makes sense. Indeed the tests were producing warnings. I actually fixed a couple of genuine `FutureWarning` recently.